packages/material-ui-system/src/index.d.ts

```diff
@@ -262,6 +263,424 @@ export const visuallyHidden: React.CSSProperties;
 
 // eslint-disable-next-line @typescript-eslint/naming-convention
 export function unstable_getThemeValue(prop: string, value: any, theme: object): any;
+
+// sx prop
+
+/**
+ * CSS typings
+ */
+
+export type StandardCSSProperties = CSS.PropertiesFallback<number | string>;
```

Comment:
The jsdoc is part of `StandardCSSProperties` now. But the comment isn't descriptive at all. What did you want to say with "CSS typings"?

Reply:
Sorry, this made sense when the typings were part of `Box.d.ts` so that I can extract the ones referring to the CSS and the ones specific to the Box component. Will remove that comment. Good catch!